jbpm-runtime-manager/src/main/java/org/jbpm/runtime/manager/impl/lock/SerializableRuntimeManagerLockStrategy.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jbpm.runtime.manager.impl.lock;
+
+import org.jbpm.runtime.manager.spi.RuntimeManagerLock;
+
+
+public class SerializableRuntimeManagerLockStrategy extends AbstractRuntimeManagerLockStrategy {
```

Comment:
Shouldn't it implement "Serializable" or is it not necessary?

Reply:
serializable is in the sense of requests not really about passsivation.